src/audio-worklet/free-queue/example/js/sink_processor.js

```diff
@@ -0,0 +1,26 @@
+import { FreeQueue } from './freequeue.mjs';
+
+class SinkProcessor extends AudioWorkletProcessor {
+    constructor(options) {
+        super();
+        this.fq = options.processorOptions.fq;
+        this.state = options.processorOptions.state;
+        Object.setPrototypeOf(this.fq, FreeQueue.prototype)
+        this.time = 0;
+    }
+
+    process(inputs, outputs) {
+        const output_buffer = outputs[0];
```

Comment:
The output "buffer" is technically not correct. The `output` ot `output0` is actually fine.